pkg/artifactory/resource/security/resource_artifactory_api_key.go

```diff
@@ -22,6 +22,11 @@ func ResourceArtifactoryApiKey() *schema.Resource {
 		CreateContext: resourceApiKeyCreate,
 		ReadContext:   resourceApiKeyRead,
 		DeleteContext: apiKeyRevoke,
+		DeprecationMessage: "An upcoming version will support the option to block the usage/creation of API Keys (for admins to set on their platform).\n" +
+			"In September 2022, the option to block the usage/creation of API Keys will be enabled by default, with the option for admins to change it back to enable API Keys.\n" +
+			"In January 2023, API Keys will be deprecated all together and the option to use them will no longer be available.\n" +
+			"It is recommended to use scoped tokens instead - `artifactory_access_token` resource.\n" +
```

Comment:
I think this should be `artifactory_scoped_token` instead. Same for the documentation.

Reply:
Sure.